src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -58,6 +64,9 @@
     public static final Person BOB = new PersonBuilder().withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
             .withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND)
             .build();
+    public static final Person CAT = new PersonBuilder().withName(VALID_NAME_CAT).withPhone(VALID_PHONE_CAT)
+            .withEmail(VALID_EMAIL_CAT).withAddress(VALID_ADDRESS_CAT).withTelegram(VALID_TELEGRAM_CAT)
+            .withGithub(VALID_GITHUB_CAT).withTags(VALID_TAG_FRIEND).build();
```

Comment:
Is there a JSON file we have to edit here?

Reply:
No, AMY and BOB also not included in any JSON file.